packages/create-styles/src/createStyleSystem/createStyleSystem.js

```diff
@@ -134,6 +153,7 @@ export function createStyleSystem(options = defaultOptions) {
 		rootStore,
 	};
 
+	// @ts-ignore `get` is problematic and difficult to get right here... maybe warrants an @todo
```

Comment:
@sirreal Would love some help on this one, I'm having a hard time parsing through the error message coming out of this (and an even harder time trying to figure out how to fix it). I'm like 99.99999999% certain what I've done is sound but something's not adding up.